spec/validators/instrument_type_validator_spec.rb

```diff
@@ -0,0 +1,215 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe InstrumentTypeValidator, reason: 'Refactor' do
```

Comment:
What's `reason: 'Refactor'` doing?

Reply:
It was left over from when I skipped the tests. I have removed it.